src/fullscreen/fullscreen-controller.js

```diff
@@ -66,8 +66,7 @@ class FullscreenController {
     return (
       this._player.env.os.name === 'iOS' &&
       !!videoElement &&
-      !!videoElement.webkitDisplayingFullscreen &&
-      (!videoElement.webkitPresentationMode || videoElement.webkitPresentationMode === 'fullscreen')
+      (videoElement.webkitPresentationMode === 'fullscreen' || (!videoElement.webkitPresentationMode && videoElement.webkitDisplayingFullscreen))
```

Comment:
Change the videoElement.webkitDisplayingFullscreen to !!videoElement.webkitDisplayingFullscreen

Reply:
not needed - && returns boolean

Reply:
Yep, it doesn't matter just prefer it inside if